scripts/gitlab/build-unix.sh

```diff
@@ -43,11 +48,6 @@ fi
 echo "_____ Calculating checksums _____"
 for binary in $(ls)
 do
-  rhash --sha256 $binary -o $binary.sha256
-  if [ "${CARGO_TARGET}" = "armv7-linux-androideabi" ]
-  then
-    echo "> ${binary} cannot be hashed with cross-compiled binary"
-  else
-    ./parity tools hash $binary > $binary.sha3
-  fi
+  rhash --sha256 $binary -o $binary.sha256 #do we still need this hash (SHA2)?
+  rhash --sha3-256 $binary -o $binary.sha3
```

Comment:
hashes are different, is that okay?
```
% target/release/parity tools hash target/release/parity
58b49924c583b45871cac4077da8f99820a684df8cca9a81c742080e991f671d
% rhash --sha3-256 target/release/parity
95fcba97a2059bbae16f997c3ea7b6c6c0f9ef5990313f55eb8536a1b3bef9e3  target/release/parity
% rhash --sha256 target/release/parity                
19c7ac31b99f5a45e6a4236367df18467321a3fc9f492c19f80cede098d47abf  target/release/parity
```

Reply:
we can't use parity to calculate the hash for other architectures